ezflow/functional/data_augmentation/augmentor.py

```diff
@@ -64,6 +66,12 @@ def __call__(self, img1, img2, flow, valid=None):
         img1, img2, flow = spatial_transform(
             img1, img2, flow, self.crop_size, **self.spatial_aug_params
         )
+        img1, img2, flow = translate_transform(
+            img1, img2, flow, **self.affine_params
```

Comment:
Should we keep two separate affine_params? For example: `translate_affine_params` and `rotation_affine_params` .  Users might want to configure different values for `translate`, `degree` and `delta`. Let me know what your thoughts.

Reply:
I'll see if it can be done. Will add a bit more complexity in regards to the yaml file and default argument values.

Reply:
Done